disnake/app_commands.py

```diff
@@ -542,15 +604,14 @@ def to_dict(self) -> Dict[str, Any]:
 
 
 class GuildApplicationCommandPermissions:
-    """
-    Represents application command permissions in a guild.
+    """Represents application command permissions in a guild.
 
     Attributes
     ----------
     id: :class:`int`
-        The ID of the corresponding command.
+        The application command's ID.
     application_id: :class:`int`
-        The ID of your application.
+        The parent application's ID. (This is usually yout bot's ID.)
```

Comment:
I bet there are many more spelling mistakes, I'll try to review them in the next few days :)